generators/generator-base.js

```diff
@@ -2284,6 +2284,7 @@ module.exports = class extends PrivateBase {
         dest.searchEngine = config.searchEngine;
         dest.cacheProvider = config.cacheProvider;
         dest.enableHibernateCache = config.enableHibernateCache;
+        dest.reactiveSqlTestContainers = config.reactiveSqlTestContainers;
```

Comment:
Can be dropped, this would load the config if it's stored at .yo-rc.json
```suggestion
```

Reply:
If I remove it, the reactiveSqlTestContainers variable is not accessible from the templates.

Reply:
Ok, you are using to load into entity generator.
`reactiveSqlTestContainers` doesn't exists in `config`, you need to calculate it.
Updated the suggestion.